internal/service/kafka/kafka_topic_cache_test.go

```diff
@@ -35,6 +35,8 @@ func TestGetTopicCache(t *testing.T) {
 			&kafkaTopicCache{
 				internal: make(map[string]map[string]aiven.KafkaTopic),
 				inQueue:  make(map[string][]string),
+				missing:  make(map[string][]string),
+				v1list:   make(map[string][]string),
 			},
```

Comment:
I think, this might be added by an accident.

Reply:
I just discovered that unit tests are failing in v3 and this is a fix for that

Reply:
I can add it in a separate commit or PR if you prefer it this way.